libs/core/src/lib/date-picker/date-picker.component.html

```diff
@@ -71,6 +71,14 @@
             [previousButtonDisableFunction]="previousButtonDisableFunction"
             [nextButtonDisableFunction]="nextButtonDisableFunction"
         ></fd-calendar>
+
+        <div fd-bar barDesign="footer">
```

Comment:
I think it should be conditional, so we will need an `*ngIf` here, and a boolean input in `date-picker.component.ts`, something like `showTodayButton`? Probably default to false as well.  So in addition, another example showing the new input in action